python/ray/experimental/workflow/api.py

```diff
@@ -8,16 +9,42 @@
 # avoid collision with arguments & APIs
 from ray.experimental.workflow import virtual_actor_class
 from ray.experimental.workflow import storage as storage_base
+from ray.experimental.workflow.storage import Storage
+from ray.experimental.workflow import workflow_access
 
 if TYPE_CHECKING:
-    from ray.experimental.workflow.storage import Storage
     from ray.experimental.workflow.virtual_actor_class import (
         VirtualActorClass, VirtualActor)
 
 logger = logging.getLogger(__name__)
 
-# TODO(suquark): Update the storage interface (e.g., passing it through
-# context instead of argument).
+
+def init(storage: "Optional[Union[str, Storage]]" = None) -> None:
+    """Initialize workflow.
+
+    Args:
+        storage: The external storage URL or a custom storage class. If not
+            specified, ``/tmp/ray/workflow_data`` will be used.
+    """
+    if not ray.is_initialized():
+        logger.info("Initializing Ray...")
+        ray.init(address="auto")
+    workflow_access.init_management_actor()
```

Comment:
should we pass the storage url to the actor?

Reply:
then should we have something like `workflow.shutdown` to terminate the actor? and how to know the actor uses the same storage? should I check it whenever I use the actor?